utils/diffing.mli

```diff
@@ -110,3 +111,19 @@ val variadic_diff :
   test:('state -> 'l -> 'r -> ('eq, 'diff) result) ->
   update:('l, 'r, 'eq, 'diff, 'state) update ->
   'state -> 'l array -> 'r array -> ('l, 'r, 'eq, 'diff) patch
+
+
+val default_weight : _ change -> int
```

Comment:
is this function actually used?

Reply:
Not anymore, because the weights have been locally specialized to account for the `Diffing_with_keys` variant (in a commit anterior to `Diffing_with_keys`). The function itself might be useful in term of documentation.